README.md

```diff
@@ -6,69 +6,54 @@ Interested in our vision? Take a look at our [whitepaper](https://jolocom.io/wp-
 
 [![Gitter chat](https://badges.gitter.im/gitterHQ/gitter.png)](https://gitter.im/jolocom/SmartWallet)
 
-Setup for development
----------------------
+## Prerequisites
 
-Setup requires Node.JS to be installed on your computer. If you do not have it please see:
-https://nodejs.org/en/download/
+- Set-up requires [Node.js](https://nodejs.org/en/download/) to be installed on your computer.
```

Comment:
As this README is also part of git, so I'm not getting why we mention here that a old version needs a different node version. This should probably mentioned in the README of versions `<1.2.0<`. IMHO

Reply:
You make a good point, but it could be that people just come to our GitHub repo first when they have issues, no matter what version they're on. It may be useful to have here anyway as it doesn't take up that much space?